app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -144,10 +147,14 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
      * However unlike categories depictions is multi-lingual
      * Ex: key: en value: monument
      */
+    private ImageInfo imageData;
+    private int oldWidth;
+    private int newWidth=0;
```

Comment:
Please use more descriptive naming explaining old width of what. And there shouldn't be a need to initialize 0 value of int, condiering it is a member variable and default value is already 0.